src/notation/internal/notationuiactions.cpp

```diff
@@ -1111,7 +1111,7 @@ const UiActionList NotationUiActions::m_actions = {
              ),
     UiAction("get-location",
              mu::context::UiCtxNotationOpened,
-             QT_TRANSLATE_NOOP("action", "Accessibility: get location")
+             QT_TRANSLATE_NOOP("action", "Accessibility: Get location")
```

Comment:
this seems inconsoostent with...

Reply:
I've just based it on what other strings beginning with ""Accessibility:" and "Note duration:" were doing, but yes there is a wider inconsistency here. Is there a preferred style that could be used for both groups?